aspnetcore/grpc/supported-platforms.md

```diff
@@ -69,6 +69,8 @@ For information about configuring ASP.NET Core servers to run gRPC, see <xref:gr
 
 &dagger;Azure App Service doesn't support hosting gRPC over HTTP/2. gRPC-Web is a compatible alternative.
 
+See [How-to deploy a .NET 6 gRPC app on App Service](https://github.com/Azure/app-service-linux-docs/blob/master/HowTo/gRPC/use_gRPC_with_dotnet.md) for Azure App Service deployment information.
+
 Work is in-progress to improve support for gRPC with HTTP/2 in Azure App Service. For more information, see [this GitHub issue](https://github.com/dotnet/AspNetCore/issues/9020).
```

Comment:
Can we remove this?

Reply:
This should be kept. gRPC + Azure + HTTP/2 support is experimental. The GitHub issue will be updated with the latest information.

When gRPC is fully supported on App Service then the link can be removed.